react-visualizer/src/reqJson/coins23-24.json

```
@@ -0,0 +1,329 @@
+{
+    "情報科学類23年度入学": {
+        "max_certificated_credit_num": 125,
+        "min_certificated_credit_num": 125,
+        "leaf": {
+            "専門": {
+                "max_certificated_credit_num": 52,
+                "min_certificated_credit_num": 52,
+                "leaf": {
+                    "": {
+                        "max_certificated_credit_num": 52,
+                        "min_certificated_credit_num": 52,
```

```suggestion
                        "max_certificated_credit_num": 50,
                        "min_certificated_credit_num": 50,
```
五月雨式にごめんなさい
専門科目全体として要求される上限と下限は50単位になってるかと思います(勘違いだったらごめんなさい)